spec/features/idv/doc_auth/verify_info_step_spec.rb

```diff
@@ -208,6 +220,7 @@
       )
 
       visit idv_verify_info_url
+      # second rate limit event
```

Comment:
Should this be removed?

Reply:
i added it because i was confused about why the tracker was receiving `idv_rate_limited` twice, based on the name of the test. i thought it might be helpful for other people like myself, but happy to remove it if you think it's not adding value. (very few folks have to dive into the tests of other teams in the way i'm doing it right now, after all)